mobile.css

```diff
@@ -0,0 +1,49 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif
+}
+.box {
+    border-style: groove;
+    border-bottom: groove thick #803a00;
+    width: 100%;
+    /*background-color: #5faf0f;*/
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding: 10px;
+    display: flex;
+    margin: 5px;
+
+}
+.container {
+    width: 25%;
+    display: flex;
+    flex-direction: column;
```

Comment:
Hi Tariro, "flex-direction : column" may be fixed a column. When change to other devices. It may can't be auto warp.